File: WebServer/Models/Enums.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Models.Enums
{
    #region 数据状态
    public enum SharedStatus
    {
        //正常
        Normal = 1,
        //审核中
        Verify = 2,
        //删除
        Delete = 3,
    }
    #endregion

    #region 角色状态
    public enum RoleStatus
    {
        //正常
        Normal = 1,
        //审核中
        Verify = 2,
        //删除
        Delete = 3,
    }
    #endregion

    #region 管理员类型
    public enum AdminType
    {
        //超级管理员
        SuperAdmin = 1,
        //次级管理员
        SecondaryAdmin = 2
    }
    #endregion

}

File: Cilent/src/common/enums.ts
export namespace SharedEnums {
    /**
     * 管理员类型
     */
    export enum AdminType {
        SuperAdmin = 1,
        SecondaryAdmin = 2,
    }
    /**
     * 角色状态
     */
    export enum RoleStatus {
        //正常
        Normal = 1,
        //审核中
        Verify = 2
    }
    /**
     * 数据状态
     */
    export enum SharedStatus {
        //正常
        Normal = 1,
        //审核中
        Verify = 2
    }
}